### DISCUSSION TITLE
Cannot upload files in webp format.

### DISCUSSION THREAD

**Discussion by GoodManWEN** [Chevereto/Chevereto-Free#168] [Help needed]
Hi everyone,

I really like the design of chevereto and have been using community version for a long time. Recently I got a new server installed the latest version of chevereto, however following the default settings (allowing all formats of images to be uploaded), neither the administrator nor the normal account can upload files in webp format (while other formats work fine), and I got the error prompt said, "example.webp - Can't get target upload source info"

I made sure that these webp files are up to spec and I am sure I have uploaded webp files in slightly earlier version of chevereto. Is there anyone who can help with this problem? Thanks very much.

ps: For testing purposes, I added an attachment with the wrong file, and since github does not support attachments with `.webp` extension, I artificially added `.jpg`  afterwards.

![example webp](https://user-images.githubusercontent.com/44322449/155114460-cd8743ed-f5e0-4fef-98bf-8857d9a1b905.jpg)

**Comment by rodber** [comment]
> "example.webp - Can't get target upload source info"

That happens here: https://github.com/rodber/chevereto-free/blob/0dce5d537421cac616b160de391562167bda6cc6/app/lib/classes/class.upload.php#L450

Fileinfo is returning false for your file, in your server: https://github.com/rodber/chevereto-free/blob/58603172b1c4b2ace16b1cb1fb9fd74a2ed6c6ae/lib/G/functions.php#L1988

Try using our Docker, I can't respond for third-party servers.
